src/TopTen/TopTenCards.js

```diff
@@ -28,19 +30,26 @@ const TopTenCards = (props) =>{
                     title="City Image"
                 />
                 <CardContent>
-                    <Typography gutterBottom variant="h5" component="h2">
+                    <Typography>
                         {props.card.name}
                     </Typography>
                 </CardContent>
             </CardActionArea>
-            <CardActions>
-                <Button size="small" color="primary">
-                    Share
-                </Button>
-                <Button size="small" color="primary">
-                    Learn More
-                </Button>
-            </CardActions>
+            <Box
+                display="flex"
+                justifyContent="space-evenly">
+                <Box display="flex"
+                     alignItems="center">
+                    <AttachMoneyIcon/>
+                    {props.card.cost_of_living}
+                </Box>
+                <Box
```

Comment:
Refactored code so that buttons that display commute time and cost of living information on the lower portion of the card would sit side by side